### PR TITLE
Add ReadMe to describe the transformer-tools

### LINKED ISSUE #3535: Add ReadMe docs for transformer-tools repo
**Description:**
The requirement is to add the ReadMe doc for the [transformer-tools](https://github.com/ballerina-platform/transformer-tools) with how to use guide.

### PR DESCRIPTION
**Pull request by AzeemMuzammil**
Note: Review comments can be found in the PR #2 

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3535

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+As of now, to activate the transformer tools on a Ballerina package, the tool required to be imported to the relevant package package as specified below. After that, when we build the Ballerina transformer package, it'll be validated against the aforementioned constraints.
+
+```ballerina
+import ballerinax/transformer as _;
+```
+
+### Ballerina Service Generator
+Once a Ballerina package is validated, the transformer tools will generate a service where it would allow the transformer functions to be consumed through REST APIs. The parameters of the transformer function has to be passed as a JSON payload.
+
+> Note: The service for the Ballerina package would only get generated if there are no any validation errors, and the parameter types and return types of transformer function should be serializable and should be supported by the Ballerina HTTP module. The list of such Ballerina types can be found in [Ballerina HTTP module specification](https://github.com/ballerina-platform/module-ballerina-http/blob/master/docs/spec/spec.md#2344-payload-parameter).
```
Comment (praneesha) [inline README.md:84]:
```suggestion
> Note: The service for the Ballerina package would only get generated if there are no validation errors, and the parameter types and return types of the transformer function should be serializable and should be supported by the Ballerina HTTP module. The list of such Ballerina types can be found in the [Ballerina HTTP module specification](https://github.com/ballerina-platform/module-ballerina-http/blob/master/docs/spec/spec.md#2344-payload-parameter).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+
+## Build from the Source
+
+### Set up the Prerequisites
```
Comment (praneesha) [inline README.md:89]:
```suggestion
### Set up the prerequisites
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+1. OpenJDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other OpenJDK distribution)
+
+   >**Info:** You can also use [Oracle JDK](https://www.oracle.com/java/technologies/javase-downloads.html). Set the JAVA_HOME environment variable to the pathname of the directory into which you installed JDK.
+
+2. Export GitHub Personal access token with read package permissions as follows,
```
Comment (praneesha) [inline README.md:95]:
```suggestion
2. Export the GitHub Personal access token with the read package permissions as follows.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+```bash
+./gradlew clean test
+```
+
+3. To build the module without the tests:
+```bash
+./gradlew clean build -x test
+```
+
+4To publish to maven local:
```
Comment (praneesha) [inline README.md:120]:
```suggestion
4. To publish to local Maven:
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+```bash
+./gradlew clean build publishToMavenLocal
+```
+
+## Contribute to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community.
```
Comment (praneesha) [inline README.md:127]:
```suggestion
As an open-source project, Ballerina welcomes contributions from the community.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
+
+## Code of conduct
+
+All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
```
Comment (praneesha) [inline README.md:133]:
```suggestion
All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
```
This link gives a 404. Please check and add the correct URL.

**Review by praneesha — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**